### PR TITLE
feat(msteams): Create better welcome message

### PR DESCRIPTION
**Pull request by KKaempen**
Make the welcome message when a user installs to Microsoft Teams look a little prettier than before. Could still use a little bit of tweaking.

Screen Shot 2020-07-09 at 11 38 58 AM

Fixes API-1120

### CODE REVIEW

File: src/sentry/integrations/msteams/webhook.py [hunk 1]
```diff
@@ -52,10 +52,57 @@ def post(self, request):
...
+                    "size": "Medium",
+                }
+                welcome = {
+                    "type": "TextBlock",
+                    "weight": "Bolder",
+                    "size": "Large",
+                    "text": "Welcome to Sentry for Teams!",
+                    "wrap": True,
+                }
+                description = {
```
Comment (NisanthanNanthakumar) [inline src/sentry/integrations/msteams/webhook.py:69]: I think we can put these constant objects into a separate file and import when needed.

Comment (KKaempen) [reply src/sentry/integrations/msteams/webhook.py:69]: That's fair, but @scefali just suggested I get it done and refactor later.

Comment (scefali) [reply src/sentry/integrations/msteams/webhook.py:69]: @KKaempen I think I didn't explain this well. There should be a separate function for sending the message. Like `send_wecome_message`. We just don't need a function to handle sending generic messages.

Comment (KKaempen) [reply src/sentry/integrations/msteams/webhook.py:69]: Gotcha, OK. Give me a sec.

Comment (KKaempen) [reply src/sentry/integrations/msteams/webhook.py]: Changed things up, sorry about misunderstanding. Let me know if there's something else I need to change.

File: src/sentry/integrations/msteams/webhook.py [hunk 2]
```diff
@@ -52,11 +52,6 @@ def post(self, request):
                     absolute_uri("/extensions/msteams/configure/"),
                     signed_params,
                 )
-                # TODO: Better message
-                payload = {
-                    "type": "message",
-                    "text": url,
-                }
-                client.send_message(team_id, payload)
+                client.send_welcome_message(team_id, url)
```
Comment (scefali) [inline src/sentry/integrations/msteams/webhook.py:55]: @KKaempen  I would move the url generation inside of `send_welcome_message`

**Review by scefali — APPROVED** [review]

**Pull request merged**